pkg/utils/utils.go

```diff
@@ -73,6 +73,7 @@ func GetPodNamespacedName(podName, podNamespace string) string {
 }
 
 func GetPodIdentifier(podName, podNamespace string) string {
+	podName = strings.Replace(podName, ".", "_", -1)
```

Comment:
nit: since this can be confusing for a customer, I wonder if we should do something like:
```
if strings.Contains(podName, ".") {
   log.Info("Replacing '.' character with '_' for pod pin path.")
    podName = strings.Replace(podName, ".", "_", -1)
}
```

Also, I see that `GetPodIdentifier` is called in multiple places, like the RPC handler here: https://github.com/aws/aws-network-policy-agent/blob/main/pkg/rpc/rpc_handler.go#L50. We should confirm with @jayanthvn and @achevuru whether each caller needs the "." name or "_" name.

Reply:
Ack, @achevuru had mentioned that `GetPodIdentifier()` would be a good place to perform the replacement but I'll still ping them for their final approval. and yeah I'll add a logging statement in there too